pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/CreateUserActivityUploadRoute.java

```diff
@@ -148,7 +147,7 @@ private long bytesToMbs(long maxFileSize) {
         return maxFileSize / MB_IN_BYTES;
     }
 
-    private String getCurrentDate() {
-        return new SimpleDateFormat("yyyy_MM_dd").format(new Date());
+    private static String getCurrentTimestamp() {
+        return new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss").format(new Date());
```

Comment:
You know what: let's add milliseconds to just virtually eliminate chance of name collision. They will be able to sort stuff like they want in the same way.

Reply:
Let's use nanoseconds then :)